imageio/core/util.py

```diff
@@ -495,8 +497,10 @@ def resource_dirs():
     in the future.
     """
     dirs = []
-    # Resource dir baked in the package
-    dirs.append(os.path.abspath(os.path.join(THIS_DIR, '..', 'resources')))
+    # Resource dir baked in the package.
+    # (The directory returned by `pkg_resources.resource_filename`
+    # also works with eggs.)
+    dirs.append(pkg_resources.resource_filename("imageio", "resources"))
```

Comment:
`pkg_resources` comes with setuptools, right? Can we please make it optional? I want to avoid cases like [this](https://stackoverflow.com/questions/7446187/no-module-named-pkg-resources).

E..g.
```py
try:
   import pkg_resources
except ImportError:
    pkg_resources = None

...

if pkg_resources:
    use pkg_resources
else:
   use the old line
```